clients/cli/src/copy.rs

```diff
@@ -24,75 +25,232 @@ pub fn copy(path: PathBuf, import_dest: &str, edit: bool) {
     });
 
     if metadata.is_file() {
-        let content_to_import = fs::read_to_string(&path).unwrap_or_else(|err| {
-            exit_with(
-                &format!("Failed to read file: {}", err),
-                COULD_NOT_READ_OS_FILE,
-            )
-        });
+        copy_file(&path, import_dest, edit, false)
+    } else {
+        recursive_copy_folder(&path, import_dest, true);
+        exit_with(&format!("imported folder to: {}", import_dest), SUCCESS)
```

Comment:
The norm with this sort of thing is that they just exit silently when they succeed, think about `mv` / `cp` from your system.